<launch>
    <machine name="host" address="localhost" />

   <!-- joy node -->
    <node machine="host" pkg="joy" type="joy_node" name="teleopJoy" output="screen">
      <param name="dev" type="string" value="/dev/input/js0" />
      <param name="deadzone"          value="0.12" />
      <param name="autorepeat_rate"   value="1.0" />
      <param name="coalesce_interval" value="0.0" />
    </node>

    <!-- define buttons & Axes for HORI Pad/PS3 Pad-->
    <param name="axis_linear"   value="3"   type="int" />
    <param name="axis_angular"  value="0"   type="int" />
    <param name="scale_linear"  value="10.0" type="double" />
    <param name="scale_angular" value="60.0" type="double" />

    <node machine="host" pkg="rc_joystick" type="rc_joystick" name="rc_joystick">	
    </node>

    <node machine="host" pkg="rosserial_python" type="serial_node.py" name="serial_node">
      <param name="port" value="/dev/ttyACM0"/>
    </node>

    <node machine="host" pkg="usb_cam" type="usb_cam_node" name="usb_cam_node">
      <param name="image_width" value="1280"/>
      <param name="framerate" value="30"/>
      <param name="pixel_format" value="mjpeg"/>
    </node>
</launch>
